### PR TITLE
feat: migrate to jest

### PR DESCRIPTION
**Pull request by snitin315**
TODO:

- [x] Fix failing tests
- [x] Remove mocha
- [x] Remove Should

### CODE REVIEW

**Comment by alexander-akait** [comment]
Looks very good, did you look why coverage is now only `81%`? I think we have a lot of tests so our coverage should be more...

**Comment by snitin315** [comment]
@alexander-akait Yes, I'm migrating tests one by one. Once all the tests move to jest, coverage will be more.

File: package.json [hunk 1]
```diff
@@ -27,12 +27,10 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.4",
     "husky": "^6.0.0",
+    "jest": "^27.5.1",
```
Comment (snitin315) [inline package.json:30]: Using jest v27 because it is compatible with node v10. We can upgrade it later.

**Comment by snitin315** [comment]
@alexander-akait I have migrated all the tests. I think this is ready.

**Review by alexander-akait — APPROVED** [review]

**Pull request merged**